src/unwind_dwfl.cc

```diff
@@ -270,30 +285,38 @@ static DDRes add_dwfl_frame(UnwindState *us, const Dso &dso, ElfAddress_t pc,
   return add_frame(symbol_idx, map_idx, pc, us);
 }
 
+static DDRes check_for_jitdump_symbol(UnwindState *us, ElfAddress_t pc,
+                                      std::string_view jitdump_path,
+                                      bool &symbol_found) {
+  assert(!jitdump_path.empty());
+  SymbolHdr &unwind_symbol_hdr = us->symbol_hdr;
+  SymbolTable &symbol_table = unwind_symbol_hdr._symbol_table;
+  RuntimeSymbolLookup &runtime_symbol_lookup =
+      unwind_symbol_hdr._runtime_symbol_lookup;
+  SymbolIdx_t symbol_idx =
+      runtime_symbol_lookup.get_or_insert(us->pid, pc, symbol_table);
+  symbol_found = symbol_idx == -1 ? false : true;
+  if (symbol_found) {
+    return add_frame(symbol_idx, -1, pc, us);
```

Comment:
I think this is probably right for now.  I can imagine a point in the future where we have some mapping information to pass, but that point isn't now.